src/util/transitTypeHelper.ts

```diff
@@ -17,4 +17,7 @@ export default class TransitTypeHelper {
             return TransitType.NOT_FOUND;
         }
     }
+    public static convertTransitTypeCodesToTransitTypes = (types: string[]) => {
+        return types.map(t => TransitTypeHelper.convertTransitTypeCodeToTransitType(t));
+    }
```

Comment:
Hmm, I'm not sure if it is worth creating an extra helper for just looping through array.

I would use:

const transitTypes = properties.transittypes.split(',').map(TransitTypeHelper.convertTransitTypeCodeToTransitType);

and remove this helper.

Reply:
I sort of disagree. Feels cleaner having this as a static helper. It has a single purpose and we will probably end up needing it elsewhere too.

Reply:
For me using .map is easier to distinct than "s" letter inside function convertTransitTypeCode_s_ToTransitType

But this isn't a big deal so change it or leave it like it is.